src/main/java/com/github/dzieciou/testing/curl/Options.java

```diff
@@ -64,6 +71,14 @@ public Builder dontLogStacktrace() {
       return this;
     }
 
+    /**
+     * Configure the level at which the generated curl statement should be logged. <code>null</code> means no logging.
+     */
+    public Builder logLevel(Level level) {
```

Comment:
To keep consistent with the rest of the code:
```suggestion
    public Builder useLogLevel(Level level) {
```